app/src/main/java/nextstep/signup/ui/theme/screen/SingUpButtonView.kt

```diff
@@ -0,0 +1,35 @@
+package nextstep.signup.ui.theme.screen
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.sp
+
+@Preview
+@Composable
+private fun SingUpButtonPreView() {
+    SingUpButtonView(Modifier)
+}
```

Comment:
SingUpButtonView에 modifier는 디폴트 파라미터로 들어가있으니, 값이 필요하지 않다면 넣지 않으셔도 되어요!
```suggestion
    SingUpButtonView()
}
```

Reply:
그러네요~ 반영하도록 하겠습니다.